src/main/java/com/sap/oss/phosphor/fosstars/data/owasp/experimental/VulnerabilitiesFromOwaspDependencyCheck.java

```diff
@@ -0,0 +1,404 @@
+package com.sap.oss.phosphor.fosstars.data.owasp.experimental;
```

Comment:
Let's please move this class to `com.sap.oss.phosphor.fosstars.data.artifact.experimental` package since it collects data about an artifact.